### PR TITLE
uriparser 0.9.4

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/uriparser.rb [hunk 1]
```diff
@@ -4,8 +4,8 @@ class Uriparser < Formula
   head "https://github.com/uriparser/uriparser.git"
 
   stable do
-    url "https://github.com/uriparser/uriparser/releases/download/uriparser-0.9.3/uriparser-0.9.3.tar.bz2"
-    sha256 "28af4adb05e811192ab5f04566bebc5ebf1c30d9ec19138f944963d52419e28f"
+    url "https://github.com/uriparser/uriparser/releases/download/uriparser-0.9.4/uriparser-0.9.4.tar.bz2"
+    sha256 "b7cdabe5611408fc2c3a10f8beecb881a0c7e93ff669c578cd9e3e6d64b8f87b"
 
     # Upstream fix, will be integrated in next release
```
Comment (SMillerDev) [inline Formula/uriparser.rb:10]: Is this now fixed?

Comment (dawidd6) [reply Formula/uriparser.rb:10]: Reading the issue, it should be in this release.

Comment (chenrui333) [reply Formula/uriparser.rb]: it can be removed, and just did it.

**Review by SMillerDev — DISMISSED** [review]

File: Formula/uriparser.rb [hunk 2]
```diff
@@ -4,15 +4,8 @@ class Uriparser < Formula
   head "https://github.com/uriparser/uriparser.git"
 
   stable do
```
Comment (SMillerDev) [inline Formula/uriparser.rb:6]: Please remove the stable block if no patches are present

Comment (chenrui333) [reply Formula/uriparser.rb:6]: 👍

**Review by dawidd6 — APPROVED** [review]

**Pull request closed without merging**